OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/CreateKeyFinalFragment.java

```diff
@@ -362,6 +365,18 @@ public boolean onCryptoSetProgress(String msg, int progress, int max) {
         mCreateOpHelper.cryptoOperation();
     }
 
+    private void setBackupReminderNotification(EditKeyResult result) {
```

Comment:
I'd suggest moving this method to BackupNotificationService, passing a context and masterKeyId.